ml-agents/mlagents/trainers/torch/networks.py

```diff
@@ -265,11 +265,17 @@ def forward(
 class GlobalSteps(nn.Module):
     def __init__(self):
         super().__init__()
-        self.global_step = torch.Tensor([0])
+        self.global_step = nn.Parameter(torch.Tensor([0]), requires_grad=False)
 
     def increment(self, value):
         self.global_step += value
 
+    def set_step(self, value):
+        self.global_step[:] = value
+
+    def get_step(self):
+        return int(self.global_step.item())
+
```

Comment:
Replace with a  property `value` or `step` getter and setter. 
```
class year_graduated:
   def __init__(self, year=32):
      self.__year = year

   @property
   def Aboutyear(self):
      return self.__year

   @Aboutyear.setter
   def Aboutyear(self, a):
      self.__year = a
```

Reply:
Why did you mark this as resolved ?

Reply:
fixed them in new commit